elbepack/initvmaction.py

```diff
@@ -269,7 +392,33 @@ class AttachAction(InitVMAction):
     def __init__(self, node):
         InitVMAction.__init__(self, node)
 
-    def execute(self, _initvmdir, _opt, _args):
+    def _attach_qemu_vm(self, initvmdir):
+        """
+        Attach to QEMU initvm.
+
+        This method is using socat to connect to the unix-socket of the 
+        serial console of the initvm.
+        """
+        initvmdir = check_initvm_dir(initvmdir)
+
+        # Test if socat command is available.
+        try:
+            subprocess.run('which socat > /dev/null 2>&1', shell=True, check=True)
```

Comment:
`shutil.which()` or catch `FileNotFoundError` coming out of `subprocess.call()` below which would be even nicer.